content/knowledge_base/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,35 @@
+# Consensus algorithm concepts behind Avalanche
+
+## Snow Algorithm group of protocols
+
+A group of protocols with a strong probabilistic safety guarantee in the presence or Byzantine nodes.
+
+### Description
+The core concept of operating is execution of repeated sampling the network at random, ultimately leading to correct nodes behavior to obtain a common statement (decision, transacation value, outcome). This mechanism effectively brings the system to an irreversible state - meaning that a large potion of the network accepted a decision or a statement. So any such conflicting to that statement would be accepted with a negligible probability ε or smaller.
+
+TODO: finish
+TODO: add references
+
+### Compare with Nakamoto type of consensus:
+TODO: add
+
+### Model
+TODO: add
+
+### Design
+TODO: add
+
+
+## Leaderless Byzantine Fault Tolerance
+
+This type of consensus algorithm combines Snowball algorithm and Practical Byznatine Fault Tolerance (pBFT) algorithm. It tries to take advantage of decentralized aspect of Snowball and deterministic property of pBFT: eliminates the weakness of Snowballs's probabilistic property and pBFT reliance on a "leader".
```

Comment:
```suggestion
This type of consensus algorithm combines the Snowball and Practical Byznatine Fault Tolerance (pBFT) algorithms. It tries to take advantage of the decentralized aspect of Snowball and deterministic property of pBFT in order to eliminate the weakness of Snowballs's probabilistic property and pBFT's reliance on a "leader".
```

WDYT?

Reply:
Looks better to me, updated https://github.com/optakt/optakt.github.io/pull/13/commits/0a1cb902f3417c3bc20f9197df71d3d65a1b6421#diff-7b8cdfdde84c7352f51f3cc642d28c90f9d1283870319c81956812353289fb32R23-R26